examples/platform/silabs/efr32/wf200/host_if.cpp

```diff
@@ -541,13 +534,8 @@ static void wfx_events_task(void * p_arg)
                                     pdTRUE, pdFALSE, pdMS_TO_TICKS(250)); /* 250 msec delay converted to ticks */
         if (flags & SL_WFX_RETRY_CONNECT)
         {
-            if (!retryInProgress)
-            {
-                retryInProgress = true;
-                wfx_retry_interval_handler(is_wifi_disconnection_event, retryJoin);
-                SILABS_LOG("WFX sending the connect command");
-                wfx_connect_to_ap();
-            }
+            SILABS_LOG("WFX sending the connect command");
```

Comment:
```suggestion
```

Reply:
@mkardous-silabs  all the logging will be fixed as a separate PR which is currently being worked on